src/unistd.rs

```diff
@@ -114,8 +114,10 @@ pub fn chdir<P: ?Sized + NixPath>(path: &P) -> Result<()> {
 #[inline]
 pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<uid_t>, group: Option<gid_t>) -> Result<()> {
     let res = try!(path.with_nix_path(|cstr| {
-        // We use `0 - 1` to get `-1 : {u,g}id_t` which is specified as the no-op value for chown(3).
```

Comment:
I remember asking for the comment back in https://github.com/nix-rust/nix/pull/274#discussion_r53517520 so... :-)

(I think it warrants the comment because it's a weird thing to do.)